src/core/qgsmaplayer.h

```diff
@@ -882,6 +882,20 @@ class CORE_EXPORT QgsMapLayer : public QObject
      */
     bool isRefreshOnNotifyEnabled() const { return mIsRefreshOnNofifyEnabled; }
 
+    /**
+     * Set last rendering time.
+     *
+     * \note not available in Python bindings
```

Comment:
Instead of just hiding this from python, I'd mark both methods as non stable API with a 

    ///@cond PRIVATE
   ...
    ///@endcond

block

Reply:
I don't think that makes any sense from Python (the amount of what is exposed in Python in general scares me), as the only valid user is the rendering engine, so perhaps do both: hide from Python + ///@cond PRIVATE . Unless the later implies the former ?

Reply:
Sorry, that's what I meant. Skip from python, but also hide from the api docs with the cond block.

Reply:
>  Unless the later implies the former ?

No, it doesn't. You still need a SIP_SKIP or #ifndef SIP_RUN